azure-mgmt-eventhub/tests/test_azure_mgmt_eventhub.py

```diff
@@ -150,16 +148,13 @@ def test_eh_eventhub_curd(self, resource_group, location):
         # Delete the created eventhub
         geteventhubresponse = self.eventhub_client.event_hubs.delete(resource_group_name, namespace_name, eventhub_name)
 
-        # Delete the create namespace
-        deletenamespace = self.eventhub_client.namespaces.delete(resource_group_name, namespace_name, None, True)
+        time.sleep(60)
```

Comment:
Why is this wait? This call is not LRO? Doing that in the test means that you just added 60 seconds to my CI...
You should change this, some solutions:
- If this is a LRO call, fix the Swagger 
- Do a GET until you get a 404.
- Add a "if self.is_live" at least to do not wait when you are in offline mode, since it's pointless to wait of you are not waiting for Azure.

Reply:
my bad, removed the sleep.